Notation.Plugin.AzureKeyVault/KeyVault/KeyVaultClient.cs

```diff
@@ -115,30 +117,36 @@ private static KeyVaultMetadata ParseId(string id)
         {
             if (string.IsNullOrEmpty(id))
             {
-                throw new ArgumentNullException(nameof(id), "Id must not be null or empty");
+                throw new ValidationException("Input passed to \"--id\" must not be empty");
             }
 
             var uri = new Uri(id);
             // Validate uri
-            if (uri.Segments.Length != 4)
+            if (uri.Segments.Length < 3 || uri.Segments.Length > 4)
             {
-                throw new ValidationException(INVALID_INPUT_ERROR_MSG);
+                throw new ValidationException("Invalid input passed to \"--id\". Please follow this format to input the ID \"https://{vault-name}.vault.azure.net/certificates/{certificate-name}\" or \"https://{vault-name}.vault.azure.net/certificates/{certificate-name}/{certificate-version}\"");
```

Comment:
how about:
```suggestion
                throw new ValidationException("Invalid input passed to \"--id\". Please follow this format to input the ID \"https://{vault-name}.vault.azure.net/certificates/{certificate-name}/[certificate-version]\"");
```

Reply:
Technically, that shouldn't be a problem. Because user of notation-akv-plugin does not use the plugin directly, they are users of Notation CLI. In Notation CLI, we use the same syntax, try `notation sign --help`, you will see `<>`, `[]` in the usage.

Reply:
Given above, we might need to consider:
`\"https://<vault-name>.vault.azure.net/certificates/<certificate-name>/[certificate-version]\"`
to be align with Notation CLI.

Reply:
Updated.